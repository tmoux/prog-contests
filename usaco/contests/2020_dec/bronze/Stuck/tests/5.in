50
N 30 33
E 60 63
N 81 99
E 59 73
N 68 90
N 26 66
E 84 76
E 36 7
N 56 79
E 87 12
E 72 59
N 42 87
E 1 13
E 21 55
N 99 21
E 39 11
E 67 5
E 27 50
E 58 20
E 22 69
E 50 25
N 40 42
E 13 98
E 9 81
N 55 94
E 79 84
N 54 26
N 44 2
E 89 72
E 53 48
N 46 68
N 0 46
E 97 49
E 63 97
N 92 86
N 52 96
N 88 57
N 28 27
E 48 56
E 47 38
N 51 35
E 4 29
N 64 85
N 77 0
E 95 43
E 82 40
N 32 61
E 96 67
N 57 14
E 5 58
