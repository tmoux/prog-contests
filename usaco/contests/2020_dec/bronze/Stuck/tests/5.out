17
28
Infinity
29
7
32
Infinity
8
Infinity
Infinity
16
Infinity
76
19
19
38
10
13
Infinity
4
49
56
29
17
Infinity
Infinity
22
9
Infinity
Infinity
Infinity
Infinity
Infinity
29
Infinity
Infinity
19
23
Infinity
4
21
24
12
20
Infinity
Infinity
37
Infinity
11
35
